util/memory_access_tracker/memory_access_tracker.c

```diff
@@ -0,0 +1,239 @@
+#include <stdio.h>
```

Comment:
Above this line, please add a copyright statement to formally declare this open source with you as the author. You only nejed this for the .c and .h files. For a model, see:
`util/gdb-dmtcp-utils`